### ISSUE TITLE
Create a documentation page about installing Backdrop in different languages

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-ops/backdropcms.org#523]
In the installer, we link to http://drupal.org/localize for more information on installing Backdrop in different languages. As part of [Switch the translation links in the installer from drupal.org to backdropcms.org](https://github.com/backdrop/backdrop-issues/issues/3560), we want to link to our own documentation page, but such a page doesn't exist on backdropcms.org.

**Comment by olafgrabienski** [comment]
While the drupal.org page [Install Drupal in another language](url) is a sub page of the "Installing Drupal 7" pages, the Backdrop [Installation instructions](https://backdropcms.org/installation) are all on *one* page. On this page, installing BD in another language is mentioned as part of the "Run the installer" steps:

> 1. Choose language
> If you want to install using a language other than English, click the *Learn how to install Backdrop in other languages* link.

Suggestions:

- remove the hint to the *Learn how to ...* link
- point to more information on the same page instead (see next suggestion)
- add the install in another language documentation in a separate section, or as part of the "Troubleshooting / Additional steps" section

(Mid-term it might be useful to split the *Installation instructions" in sub pages, as Drupal does.)

**Comment by olafgrabienski** [comment]
Here's a draft of the documentation text, only the needed steps, wording to be improved:

**Add a language before installing Backdrop**

- Download the language file from the [translation server](https://localize.backdropcms.org).
- Move the downloaded .po file inside the directory `files/translations`.
- Install Backdrop as per usual.

**Add a language after installing Backdrop**

- Go to **Functionality > List Modules**, and ensure that the *Locale* module is enabled.
- Go to **Configuration > Regional and language > Languages**, click the "Add language" link, select your language, and click the "Add language" button.
- Download the language file from the [translation server](https://localize.backdropcms.org).
- Go to **Configuration > Regional and language > User interface translation > Import**. Upload and import the language file using the form.

**Comment by olafgrabienski** [comment]
@klonos @jenlampton I'd love to get feedback on the documentation suggestions above. When we have an agreement about the text, can I edit the documentation pages (permission-wise), or do you want to do it?

Note: We first need the documentation on b.org before I can file a pull request for https://github.com/backdrop/backdrop-issues/issues/3560.

**Comment by jenlampton** [comment]
@olafgrabienski  I have added your content to a page on b.org:
https://backdropcms.org/user-guide/install-backdrop-another-language

You should have permission to edit this page.

**Comment by herbdool** [comment]
@olafgrabienski maybe it should be clearer that if the Backdrop server doesn't have a complete translation they can download ones from Drupal for some coverage.

Also, not sure where to put info on how to contribute translations.

**Comment by olafgrabienski** [comment]
@herbdool Good points, will consider them when I edit the page, hopefully tomorrow.

**Comment by olafgrabienski** [comment]
I've updated the page https://backdropcms.org/user-guide/install-backdrop-another-language, primarily added an intro text. (Not yet visible when not logged in. I have cleared the "Page and else" cache, though.)

@herbdool In my opinion, information about Drupal language files and about contributing to Backdrop translations should live on the translation site. It's actually there, in the *Welcome* text, but still very basic and definitely to be improved.

(More) information about Drupal language files should e.g. also displayed on https://localize.backdropcms.org/translate and sub pages.

The contribution workflow is still not very clear. When it is, I'll add that information.

**Comment by olafgrabienski** [comment]
@herbdool I forgot to mention that there is also a "How to contribute translations" documentation issue: https://github.com/backdrop-ops/backdropcms.org/issues/507

**Comment by olafgrabienski** [comment]
In https://github.com/backdrop/backdrop-issues/issues/3560#issuecomment-481450012 @quicksketch suggested to remove the `/user-guide` part from the URL of the page. Instead of

`https://backdropcms.org/user-guide/install-backdrop-another-language`

the URL would be:

`https://backdropcms.org/install-backdrop-another-language`

**Comment by olafgrabienski** [comment]
@jenlampton or @klonos - Can you change the URL alias for the page to 'install-backdrop-another-language`? I don't have the permissions.

**Comment by jenlampton** [comment]
How's this?
https://backdropcms.org/installing-in-other-languages

**Comment by olafgrabienski** [comment]
> https://backdropcms.org/installing-in-other-languages

Looks good, thank you!

**Comment by docwilmot** [comment]
Just curious, as I've never installed in another language, but 
> Add a language before installing Backdrop
Download the language file from the translation server.
Move the downloaded .po file inside the directory files/translations.
Install Backdrop as per usual.
You'll find your language on the page Configuration > Regional and language > Languages.

Doesn't this process change the installation steps at all (does Backdrop not prompt you to choose a language during install or something?) and if so, the steps should mention that. Like step 3 "you will be prompted to choose a language" etc

**Comment by olafgrabienski** [comment]
> Doesn't this process change the installation steps at all (does Backdrop not prompt you to choose a language during install or something?) and if so, the steps should mention that.

Good point! The information is on the general installation page but we should add it here as well. I'll update the page tomorrow.

**Comment by olafgrabienski** [comment]
@docwilmot Thanks for the hint yesterday. I've tested the installation process and have updated the documentation.

I've also noticed that files from the Backdrop localization server currently need to be renamed, in contrast to Drupal files (fixed for them in https://github.com/backdrop/backdrop-issues/issues/2345).

Part of the current text:

> **Add a language before installing Backdrop**
> - Download the language file from the translation server.
> - Move the downloaded `.po` file inside the directory `files/translations`.
> - Rename the file to `install.(languagecode).po`.
>   Example: Rename `backdropcms-1.12.2.es.po` to `install.es.po`.
> - Install Backdrop as per usual.
>   The installer will guide you through several steps.
>   During the first step, you will be prompted to choose a language.
> - Choose your language, and click to continue the installation.
>   During some of the next steps, the installer will import your translations.
  > 
>   After the installation you will be redirected to your site's home page.
>   You'll find your language ...
